pkg/pf/internal/providerbuilder/build_resource.go

```diff
@@ -68,12 +69,16 @@ func NewResource(args NewResourceArgs) Resource {
 			resp.State = tfsdk.State(req.Plan)
 		}
 	}
+	if args.ReadFunc == nil {
+		args.ReadFunc = func(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+			resp.State = req.State
```

Comment:
This seems like a sensible default for Reads, when the test does not care to implement it; copy the prior state as the "recently acquired" state.